app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author = current_user
+    @comment.post = Post.find(params[:post_id])
+
+    respond_to do |_format|
+      if @comment.save
+
+        redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
```

Comment:
- [ ] **[OPTIONAL]** From your code a comment is created users should be redirected to the post, but that does not happen, the app remains on the create new comment page. You may want to look into that.

  ![image](https://github.com/iamsjunaid/quill-quest/assets/36542986/4a9a26b1-99c0-417d-895a-9a34928801b4)